### PR TITLE
[Compression] Add bias correction feature for PTQ quantizer

### PR DESCRIPTION
**Pull request by Bonytu**
### Description ###
Bias correction in post-training quantization refers to the process of adjusting and fine-tuning the quantized model's weights and biases to reduce the discrepancies between the quantized model's predictions and the original full-precision model's predictions. Post-training quantization is a technique used to reduce the memory and computation requirements of deep learning models by converting the model's parameters, such as weights and biases, into lower-precision representations (e.g., from 32-bit floating-point numbers to 8-bit integers). This can lead to some loss of accuracy due to the reduced numerical precision. Bias correction aims to mitigate this accuracy loss by correcting the systematic errors introduced during the quantization process, ultimately improving the overall performance of the quantized model.

#### Test Options ####
  - [ ] fast test
  - [ ] full test - HPO
  - [ ] full test - NAS
  - [ ] full test - compression

### Checklist ###
  - [ ] test case
  - [ ] doc

### CODE REVIEW

File: nni/contrib/compression/base/wrapper.py [hunk 1]
```diff
@@ -36,7 +37,7 @@
 
 
 class ModuleWrapper(torch.nn.Module):
-    def __init__(self, module: torch.nn.Module, module_name: str, config: Dict[str, Dict[str, Any]] | None = None, \
+    def __init__(self, module: torch.nn.Module, module_name: str, config: Dict[str, List[Dict[str, Any]]] | None = None, \
```
Comment (QuanluZhang) [inline nni/contrib/compression/base/wrapper.py:40]: it would be better to update docstring to explain the value field of `config`

**Comment by QuanluZhang** [comment]
could you briefly explain what is bias correction feature in this pr's description?

File: nni/contrib/compression/quantization/lsqplus_quantizer.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+    Parameters
+    ----------
+    model
+        Model to be quantized.
+    config_list
+        A list of dict, each dict configure which module need to be quantized, and how to quantize.
+        Please refer :doc:`Compression Config Specification </compression/config_list>` for more information.
+    evaluator
+        TODO: {evaluator_docstring}
```
Comment (QuanluZhang) [inline nni/contrib/compression/quantization/lsqplus_quantizer.py:37]: still "TODO" here?

**Comment by Bonytu** [comment]
> could you briefly explain what is bias correction feature in this pr's description?

Bias correction in post-training quantization refers to the process of adjusting and fine-tuning the quantized model's weights and biases to reduce the discrepancies between the quantized model's predictions and the original full-precision model's predictions. Post-training quantization is a technique used to reduce the memory and computation requirements of deep learning models by converting the model's parameters, such as weights and biases, into lower-precision representations (e.g., from 32-bit floating-point numbers to 8-bit integers). This can lead to some loss of accuracy due to the reduced numerical precision. Bias correction aims to mitigate this accuracy loss by correcting the systematic errors introduced during the quantization process, ultimately improving the overall performance of the quantized model.

**Review by QuanluZhang — APPROVED** [review]

**Review by J-shang — APPROVED** [review]
